### ISSUE TITLE
Implement UI for new AdSense setup site components

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#4764]
This issue is for actually implementing the UI (and a bit off extra logic) for the enhanced AdSense setup flow main components that were introduced in #4760.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The `SetupAccountSite` component from #4760 should have its UI implemented. The existing conditions in that component (that currently just return dummy content) should be kept, but now return the actual UI needed, as follows:
    * All UI variations should follow the same high-level markup (e.g. copy it from an existing component like `SetupSiteAdd` or `SetupAccountNoClient`), each one with
        * a heading
        * the `ErrorNotices` component (which shows errors just if there are some)
        * a main body text in a `p`
        * the `SetupUseSnippetSwitch` from #4761 (which is displayed conditionally by itself)
        * either a CTA `<Button>` or `<Link>` (with technically both being links, but the appearance would differ)
    * For the `NEEDS_ATTENTION` case, introduce a new component `SetupAccountSiteNeedsAttention` with the following content:
        * Heading: _Your site isn’t ready to show ads yet_
        * Body text: _You need to fix some things with this site before we can connect Site Kit to your AdSense account._
        * `<Button>`:
            * Text: _Review site in AdSense_
            * URL: Result of the `getServiceAccountManageSitesURL` selector
            * Clicking the button should fire a GA tracking event (category: `${ viewContext }_adsense`, name: `review_site_state`, label: `needs_attention`), and the link URL should open in a new window.
    * For the `REQUIRES_REVIEW` case, introduce a new component `SetupAccountSiteRequiresReview` with the following content:
        * Heading: _Your site requires review_
        * Body text: _To start serving ads, your site needs to be approved first. Go to AdSense to request the review._
        * `<Button>`:
            * Text: _Request review in AdSense_
            * URL: Result of the `getServiceAccountManageSitesURL` selector
            * Clicking the button should fire a GA tracking event (category: `${ viewContext }_adsense`, name: `review_site_state`, label: `requires_review`), and the link URL should open in a new window.
    * For the `GETTING_READY` case, introduce a new component `SetupAccountSiteGettingReady` with the following content:
        * Heading: _Your site is getting ready_
        * Body text: _This usually takes a few days, but in some cases can take up to 2 weeks. You’ll get an email from AdSense as soon as they have run some checks on your site._
        * `<Link>`:
            * Text: _Review site in AdSense_
            * URL: Result of the `getServiceAccountManageSitesURL` selector
            * Clicking the button should fire a GA tracking event (category: `${ viewContext }_adsense`, name: `review_site_state`, label: `getting_ready`), and the link URL should open in a new window.
    * For the `READY` case (regardless of whether auto ads are enabled or not), introduce a new component `SetupAccountSiteReady`. That component should now include the check for `site.auto_ads_enabled` and whether it is `true` or `false`. It should also include a local state variable that allows storing whether the user has acknowledged that auto ads are disabled (since we want to encourage users to enable it, but not force them to).
        * If auto ads are disabled _and_ the user has not yet acknowledged that (see above state variable), the following content should be displayed:
            * Heading: _Enable auto ads for your site_
            * Body text:
                * If there is no existing tag: _To start serving ads via Site Kit, you need to activate auto ads first. Go to AdSense and enable auto ads for your site._
                * If there is an existing tag: _Site Kit recommends enabling auto ads. If your existing AdSense setup relies on individual ad units, you can proceed without enabling auto ads._
            * `<Button>`:
                * Text: _Enable auto ads_
                * URL: Result of the `getServiceAccountSiteAdsPreviewURL` selector
                * Clicking the button should fire a GA tracking event (category: `${ viewContext }_adsense`, name: `enable_auto_ads`), and the link URL should open in a new window.
            * `<Link>` (this should be a secondary action appearing on the right of the above `<Button>`):
                * **This should only be displayed if there is an existing tag.** Otherwise, the user should essentially be forced to click on the above _Enable auto ads_ link, since that is the only choice for new users.
                * Text: _Proceed without enabling auto ads_
                * URL: none, this is an _actual_ button (however with appearance like a link)
                * Clicking the button should fire a GA tracking event (category: `${ viewContext }_adsense`, name: `disable_auto_ads`), and it should set the above state variable to acknowledge that auto ads are disabled (i.e. after clicking this the user should land in the other alternative UI of this component).
        * Otherwise (i.e. auto ads are enabled _or_ the user has acknowledged that they aren't), the following content should be displayed:
            * Heading:
                * If there is no existing tag: _Your site is ready to use AdSense_
                * If there is an existing tag: _Your AdSense account is ready to connect to Site Kit_
            * Body text:
                * If there is no existing tag: _Site Kit has placed AdSense code on your site to connect your site to AdSense and help you get the most out of ads._
                * If there is an existing tag: _Connect your AdSense account to see stats on your overall earnings, page CTR, and top earning pages._
            * `<Button>`:
                * Text: _Configure AdSense_
                * URL: none, this is an _actual_ button
                * Clicking the button should complete the setup (i.e. this should use the logic that has already been scaffolded before in #4760, calling `completeAccountSetup`, `completeSiteSetup`, and `finishSetup`. It should not fire any GA event since the `finishSetup` callback already has one.
* The UI implemented here can be tested via the Storybook stories introduced in #4760, and it should be ensured it looks correct there (in case e.g. any additional data needs to be bootstrapped in the Storybook configuration).

## Implementation Brief
* Create `assets/js/modules/adsense/components/setup/v2/common/SetupAccountSiteUI.js` which exports the `SetupAccountSiteUI` functional component with the following details:
	* It should copy the same HTML markup as `assets/js/modules/adsense/components/setup/SetupSiteAdd.js` with the following changes:
		* Adding the `assets/js/modules/adsense/components/setup/v2/SetupUseSnippetSwitch.js ` which will be rendered after the `p` tag.
		* Addition of another `Button` component which will be rendered after the existing `Button` one.
		* `googlesitekit-setup-module__*` class names should be updated to `googlesitekit-adsense-setup__` and styles copied over accordingly if any.
	* It should accept the following props:
		* `heading`: String to be rendered within the `h3` tag.
		* `description`: String to be rendered within the `p` tag.
		* `primaryButton`: Object with the following properties: 
			* `href`: String to be used as the `href` attribute of the `Button` component.
			* `label`: String to be used as the `children` property of the `Button` component.
			* `onClick`: Callback function if defined when the `Button` is clicked, to be assigned to the `onClick` property of the `Button` component.
		* `secondaryButton`: Object having the same properties as `primaryButton`.
* Using the following components: `SetupAccountSiteNeedsAttention`, `SetupAccountSiteRequiresReview` and `SetupAccountSiteGettingReady`, create their corresponding files in `assets/js/modules/adsense/components/setup/v2/` which export the same functional component as their file name and with the details below:
	* They should render the `SetupAccountSiteUI` component using the above props and strings (translated) from the AC for each component.
	* They should define a `onButtonClick` function which will call `trackEvent` with the parameters defined in the AC and this function should be passed as the `onClick` property of the `primaryButton` prop.
* Create `assets/js/modules/adsense/components/setup/v2/SetupAccountSiteReady.js` which exports the `SetupAccountSiteReady` functional component with the following details:
	* It should accept the `site` object prop.
	* Add a new local state variable `acknowledgedDisabledAutoAds` with default value `false`
	* Query the `modules/adsense` data store via the `getExistingTag` selector to check if there's an existing tag.
	* Use the `SetupAccountSiteUI` component to render the strings as per the AC when :
		* `site.auto_ads` is `false` **and** `acknowledgedDisabledAutoAds` is `false` along whether there is an existing tag or not.
		* `site.auto_ads` is `true` **or** `acknowledgedDisabledAutoAds` is `true` along whether there is an existing tag or not.
* Using `assets/js/modules/adsense/components/setup/v2/SetupAccountSite.js`, render the following components with the appropriate condition:
	* `NEEDS_ATTENTION`: `SetupAccountSiteNeedsAttention`
	* `REQUIRES_REVIEW`: `SetupAccountSiteRequiresReview`
	* `GETTING_READY`: `SetupAccountSiteGettingReady`
	* `READY`: `SetupAccountSiteReady`
* Ensure the stories in `assets/js/modules/adsense/components/setup/v2/SetupMain.stories.js` are rendered properly.

### Test Coverage
* No new tests to be added.

### Within the plugin:
* Verify the existing setup for AdSense works as before (without enabling the `adsenseSetupV2` flag).
* Using the latest version of the Tester Plugin, turn on the `adsenseSetupV2 ` flag. Set the `Force AdSense account status` to `ready` and keep it so throughout the following steps. Now cycle through the different options of `Force AdSense site status` from `needs attention` to `ready` and attempt to activate/setup AdSense within the plugin for each site status option. Verify the UI is as per the AC for each site status. The `ready` and `ready-no-auto-ads` statuses need to be tested with and without a value set for `Place AdSense existing tag`. `ca-pub-123456789` could be used as a valid existing tag value. So effectively, the `READY` status in the AC has four variations that need to be tested: With and without `Auto Ads Enabled` + with and without an existing tag being present.

### Within Storybook:
* Verify the [new stories](https://google.github.io/site-kit-wp/storybook/develop/?path=/story/modules-adsense-components-setup-v2-setupmain--setup-account-site-needs-attention) added in StoryBook meet the AC. The stories added/modified are from `Account: Site - Needs Attention` to `Account: Site - Invalid Site State`. The last story is not in the AC but stems [from this comment](https://github.com/google/site-kit-wp/pull/5138#discussion_r861141367). 

## Changelog entry

* Implement UI for new AdSense setup site components.

**Comment by tofumatt** [comment]
IB ✅

**Comment by felixarntz** [comment]
@jimmymadon Replied to the question on the PR. Regarding the QA Brief, that's a bit vague, it's not very helpful to say to verify the ACs. We also have the tester plugin version 1.8.0 released now which allows mocking all these states, so I think the QA here could already be in-product. I'd advise iterating on the QA Brief with a more clear outline on how to set account status (can probably be set to `ready` no matter what) and site status (go through all the relevant statuses) in the tester plugin and what to verify. The [user-facing behavior section of the design doc](https://docs.google.com/document/d/1vNpMW4iED2scWOjpF8SKGoM8z6GUjX-WbhMpmAKeieo/edit#heading=h.rsi8f1yufqjk) may also help as a reference on what content to verify.

**Comment by jimmymadon** [comment]
@felixarntz Ah thanks for pointing this out! Since the AC and IB both hinted at testing this solely in Storybook, I had a feeling that there might be other related issues that need to be completed for it all to be tested in-product. And the new stories in storybook which I linked to are almost a 1:1 mapping of the ACs - hence my rather brief comment for the QAB! Will update this as suggested.

**Comment by wpdarren** [comment]
### QA Update: ❌ 

@jimmymadon I have two observations:

1. When the `Force AdSense site status` is set to `Ready` with auto ads enabled, when I click on the 'Configure AdSense' button nothing happens. No error in the console. When I go to Site Kit settings though it's telling me that AdSense is successfully connected.  Also the account status in settings appears as `Your site isn’t ready to show ads yet`. I am assuming this is not expected going on the AC.  

2. When the `Force AdSense site status` is set to `Ready No Auto Ads` and there are no existing tags, the `Proceed without enabling Auto Ads` link does not appear.  The link does appear when there is an existing tag. 

3. This is a general question because I don't feel it is mentioned in the AC (might be in another ticket.) When the `Force AdSense site status` is set to `Ready No Auto Ads` and I click on the `Enable Auto Ads` button, at this stage what should load? The link URL is https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com. On a site that has a Google account with AdSense ready, it redirects me to the Ad Settings Preview, and I do not see any option to enable auto ads, so it's a little confusing. 

**Verified:**

- The existing setup for AdSense works as before, without enabling the `adsenseSetupV2` flag.  I tested this on a site using the tester plugin to activate an account with demo data.  I also tested it on my live site that is approved with AdSense and ads were displayed and the setup was as before. 

_Needs attention:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

![image](https://user-images.githubusercontent.com/73545194/166230453-2aceed32-8548-464a-acdb-4b77e96ce9b1.png)

_Needs review:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

![image](https://user-images.githubusercontent.com/73545194/166231329-3713c9a5-a370-42ee-b44d-20a812ebc62e.png)

_Getting ready:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

![image](https://user-images.githubusercontent.com/73545194/166232253-3b226a2f-4a62-4c61-bd2b-d5c6f606beb7.png)

_Ready: no auto ads, no existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.
 
![image](https://user-images.githubusercontent.com/73545194/166235024-8b57fcb9-ce4b-4b6f-882c-5c4197978e8d.png)

_Ready: no auto ads, existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.

![image](https://user-images.githubusercontent.com/73545194/166238937-bfbb959e-ba79-4c44-9281-3a6ba07dedad.png)

**Comment by jimmymadon** [comment]
@wpdarren Thanks for the thorough QA as always! 

1. For observation 1, I have created a [follow-up PR](https://github.com/google/site-kit-wp/pull/5163). The user should be redirected to the SK Dashboard when clicking the Configure AdSense button in this case. This tiny bit of functionality was broken / not fully implemented in preceding issues - but with the UI now complete, it makes sense to tie this end up! 👍 
2. Your observation seems to be in line with the AC (the Ready case does have many pathways and conditions!):
>This should only be displayed if there is an existing tag. Otherwise, the user should essentially be forced to click on the above Enable auto ads link, since that is the only choice for new users.  
3. I believe this behaviour is happening because the tester plugin is 'forcing' the site status to be 'Ready with _Auto Ads Disabled_'. So even when you go back to the site, it still shows the same components. Are you able to check this on your live testing site?

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@jimmymadon Re. no.2 of my list of observations. 

> Your observation seems to be in line with the AC (the Ready case does have many pathways and conditions!):
This should only be displayed if there is an existing tag. Otherwise, the user should essentially be forced to click on the above Enable auto ads link, since that is the only choice for new users.

Good spot on the AC!  For new users we are forcing them to enable auto ads, why is that?  I'm thinking about it as a user of Site Kit with AdSense and personally, I don't like where the ads are placed. When we have an existing tag we are allowing them to move on without auto ads.  I am happy to move on and forget this, but maybe @felixarntz could clarify the reasoning?  🤔

I will re-test issue 1 and use my live site for issue 3.  Thanks!

**Comment by wpdarren** [comment]
### QA Update:  ⚠️ 

@jimmymadon @felixarntz Apologies, I have a few more observations, which I'd like clarification on:

Re. no. 1 of my [list of observations](https://github.com/google/site-kit-wp/issues/4764#issuecomment-1114898401). 

- Confirmed that issue 1 is fixed. When I click on the 'Configure AdSense' button, I am redirected back to the Site Kit Dashboard and the notification appears confirming that AdSense is connected.  Why is the button titled 'Configure AdSense' I feel this is confusing. Maybe it should be 'Complete setup' instead?  If this is out of scope, I can create a ticket. 

Re. no.2 of my list of observations.

> Your observation seems to be in line with the AC (the Ready case does have many pathways and conditions!):
This should only be displayed if there is an existing tag. Otherwise, the user should essentially be forced to click on the above Enable auto ads link, since that is the only choice for new users.

Good spot on the AC! For new users we are forcing them to enable auto ads, why is that? I'm thinking about it as a user of Site Kit with AdSense and personally, I don't like where the ads are placed. When we have an existing tag we are allowing them to move on without auto ads. I am happy to move on and forget this, but maybe Felix could clarify the reasoning? 🤔

Re. No.3 of my list of observations. 

> I believe this behaviour is happening because the tester plugin is 'forcing' the site status to be 'Ready with Auto Ads Disabled'. So even when you go back to the site, it still shows the same components. Are you able to check this on your live testing site?

See a weird UX/UI when using my live site.  I click to `enable live ads`, go to AdSense and then click for auto ads and save.  When I come back to my site though I am unsure what to do. There's no prompt.  When I refresh, the `Your site is ready to use AdSense` message is loaded, so that's good.  The user won't know to refresh though. 

 I feel there should be a prompt to refresh so that the page updates.

Thoughts? 

**Verified:**

- The existing setup for AdSense works as before, without enabling the `adsenseSetupV2` flag.  I tested this on a site using the tester plugin to activate an account with demo data.  I also tested it on my live site that is approved with AdSense and ads were displayed and the setup was as before. 

_Needs attention:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Needs review:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Getting ready:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Ready: no auto ads, no existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.
 

_Ready: no auto ads, existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.

<details>
 <summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/73545194/166230453-2aceed32-8548-464a-acdb-4b77e96ce9b1.png)

![image](https://user-images.githubusercontent.com/73545194/166231329-3713c9a5-a370-42ee-b44d-20a812ebc62e.png)

![image](https://user-images.githubusercontent.com/73545194/166232253-3b226a2f-4a62-4c61-bd2b-d5c6f606beb7.png)

![image](https://user-images.githubusercontent.com/73545194/166235024-8b57fcb9-ce4b-4b6f-882c-5c4197978e8d.png)

![image](https://user-images.githubusercontent.com/73545194/166238937-bfbb959e-ba79-4c44-9281-3a6ba07dedad.png)

</details>

**Comment by wpdarren** [comment]
I've sent this back to execution because feel that this is where it belongs right now due to the issues [reported above](https://github.com/google/site-kit-wp/issues/4764#issuecomment-1116192374).

**Comment by wpdarren** [comment]
Thanks @felixarntz for the updates, appreciated.

> The buttons in the setup flows to complete the setup always say "Configure ", that's why it is like that. I agree though that can be misleading, but I would advise to rethink the wording of these buttons across all module setups then, not just this one. Definitely supportive of opening an issue, but it shouldn't be in this epic and apply to all modules rather than just AdSense.

I will create an Asana ticket to remind me to create a more thorough Github ticket for this once we have the release out of the way. I'd not realised it said configure on other modules. 

> Can you clarify this particular observation? Only sites without existing tags should be forced to enable auto ads - sites with an existing tag should be able to click the secondary link to bypass this, regardless of what their setting on the AdSense side is. If that is not the case right now, this is incorrect and needs to be fixed.

Yes, this is the correct behaviour. If I do not have an existing tag then I am forced to auto ads. If there is an existing tag then there is a link to bypass the auto ads.  I now understand  more about why we are forcing auto ads within the plugin.  This is not an issue, so can move on. Thanks for the explanation. 

> This probably happened because you took "too little" time to enable auto ads in the AdSense frontend. We have that functionality that refreshes the AdSense setup flow once you've come back from another tab, and there is a minimum timeout. It looks like that may be set a bit too high. Can you test this again while waiting a bit longer, just to know whether it's in fact that? In any case, this is a problem we need to address, but let's do it in a separate issue. Can you please open one? We can treat that as a known issue for the bug bash, if we don't have it fixed by then.

Yes, this is an issue. 

I am taking my time in all the steps and when I go back to the tab with Site Kit after enabling auto ads, nothing happens and unable to complete the set up of AdSense.   I will create an Asana ticket to create a Github ticket with more details once release is out of the way.  ASI bug bash isn't for a little while yet, so in the meantime, will move this to approved.

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- The existing setup for AdSense works as before, without enabling the `adsenseSetupV2` flag.  I tested this on a site using the tester plugin to activate an account with demo data.  I also tested it on my live site that is approved with AdSense and ads were displayed and the setup was as before. 

_Needs attention:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Needs review:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Getting ready:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/sites/my-sites?source=site-kit&authuser=myemail%40gmail.com` The tracking is also as per the AC.

_Ready: no auto ads, no existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.
 

_Ready: no auto ads, existing tag:_ 
Title, description, button text are per the AC.  URL opens in a new window and points to `https://www.google.com/adsense/new/u/0/pub-1234567890123456/myads/sites/preview?source=site-kit&url=instawp.xyz&authuser=myemail%40gmail.com` The tracking is also as per the AC. Testing this on a site that has an approved Google account, the page opens as expected with Ad Settings Preview.

<details>
 <summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/73545194/166230453-2aceed32-8548-464a-acdb-4b77e96ce9b1.png)

![image](https://user-images.githubusercontent.com/73545194/166231329-3713c9a5-a370-42ee-b44d-20a812ebc62e.png)

![image](https://user-images.githubusercontent.com/73545194/166232253-3b226a2f-4a62-4c61-bd2b-d5c6f606beb7.png)

![image](https://user-images.githubusercontent.com/73545194/166235024-8b57fcb9-ce4b-4b6f-882c-5c4197978e8d.png)

![image](https://user-images.githubusercontent.com/73545194/166238937-bfbb959e-ba79-4c44-9281-3a6ba07dedad.png)

</details>

**Comment by jimmymadon** [comment]
@felixarntz So in the absence of Figma mocks, I referenced other similar Module Setup screens which have a secondary CTA button, e.g. [Analytics Create Account](https://google.github.io/site-kit-wp/storybook/main/?path=/story/analytics-module-setup--create-account-scope-not-granted) and [Tag Manager Create Account](https://google.github.io/site-kit-wp/storybook/main/?path=/story/tag-manager-module-setup--set-up-a-new-account). Happy to make a follow up PR - but should I do that for just this secondary button or should we fix the others too?

**Comment by felixarntz** [comment]
@jimmymadon @aaemnnosttv Makes sense, thanks. I'll add ACs to #5184.
